doc/classes/NavigationAgent2D.xml

```diff
@@ -1,11 +1,12 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="NavigationAgent2D" inherits="Node" is_experimental="true" version="4.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
-		2D Agent used in navigation for collision avoidance.
+		A 2D agent used to pathfind to a position while avoiding obstacles.
 	</brief_description>
 	<description>
-		2D Agent that is used in navigation to reach a position while avoiding static and dynamic obstacles. The dynamic obstacles are avoided using RVO collision avoidance. The agent needs navigation data to work correctly. [NavigationAgent2D] is physics safe.
-		[b]Note:[/b] After [member target_position] is set, the [method get_next_path_position] function must be used once every physics frame to update the internal path logic of the NavigationAgent. The returned position from this function should be used as the next movement position for the agent's parent node.
+		A 2D agent used to pathfind to a position while avoiding static and dynamic obstacles. Child nodes added to it will move with the agent. Requires navigation data to work correctly.
```

Comment:
NavigationAgent does not move child nodes, it inherits Node and has no transform.